core/arch/aarch64/codec.h

```diff
@@ -30,8 +30,8 @@
  * DAMAGE.
  */
 
-#ifndef CODEC_H
-#define CODEC_H 1
+#ifndef CODEC_H_
```

Comment:
Can we assume that the first preprocessor #define in a .h file is implicitly around-whole-file and thus avoid changing any of these headers